.cirrus.yml

```diff
@@ -101,14 +113,15 @@ task:
     - env:
         CC: gcc
```

Comment:
Hm, your new `file` invocations shows that this actually builds 64-bit binaries https://cirrus-ci.com/task/5529748383203328?command=test#L212. Not sure if this is missing the `-m32` flag or if the automake `--host` option should have set that.

Reply:
Hm damn, I played a lot with this, and in the end I thought it works without the `-m32` flag. I'll have a look... 

By the way, a difference to the previous approach on Debian (what we had on Travis) is that this here doesn't use gcc-multilib because this is not compatible with installing the other s390x compiler (okay we could have two images) and I generally considered it to be more consistent: "multilib" is a thing we anyway can do only on x86(_64) and not for s390x (and arm and more archs, which we hopefully can add now by basically copy and paste).

Reply:
Ok, the issue was indeed that we set `CC` explicitly to `gcc`. When it's not set, configure finds the right `i686-linux-gn-gcc`. I've set it explicitly to `i686-linux-gnu-gcc` now.

Reply:
Yup, thanks that appears to work.